packages/clerk-js/src/ui/elements/SocialButtons.tsx

```diff
@@ -16,7 +16,7 @@ export type SocialButtonsProps = React.PropsWithChildren<{
 }>;
 
 type SocialButtonsRootProps = SocialButtonsProps & {
-  oauthCallback: (strategy: OAuthStrategy) => Promise<unknown>;
+  oauthCallback: (strategy: OAuthStrategy, captchaToken?: string) => Promise<unknown>;
```

Comment:
Should we add it as an options object ?
```suggestion
  oauthCallback: (strategy: OAuthStrategy, options: {captchaToken?: string}) => Promise<unknown>;
```

Reply:
@panteliselef this is a leftover. i'll remove this